mmv1/templates/terraform/constants/region_ssl_policy.erb

```diff
@@ -0,0 +1,34 @@
+<%# The license inside this block applies to this file
+	# Copyright 2017 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+func regionSslPolicyCustomizeDiff(_ context.Context, diff *schema.ResourceDiff, v interface{}) error {
+	profile := diff.Get("profile")
+	customFeaturesCount := diff.Get("custom_features.#")
+
+	// Validate that policy configs aren't incompatible during all phases
+	// CUSTOM profile demands non-zero custom_features, and other profiles (i.e., not CUSTOM) demand zero custom_features
+	if diff.HasChange("profile") || diff.HasChange("custom_features") {
+		if profile.(string) == "CUSTOM" {
```

Comment:
can we do an `&&` here?
```
if profile.(string) == "CUSTOM" && customFeaturesCount.(int) == 0 {...}
else if customFeaturesCount.(int) != 0 {...}
```

Reply:
I think we would then need to check the value of `profile.(string)` twice to map out the invalid cases. 
This is does seem clearer 

```
if profile.(string) == "CUSTOM" && customFeaturesCount.(int) == 0 {...}
else if profile.(string) != "CUSTOM" && customFeaturesCount.(int) != 0 {...}
```